### ISSUE TITLE
dbus-send syntax, gnome-panel question

### ISSUE THREAD

**Issue by brandonwillard** [qtile/qtile#810]
Just tried the Gnome setup from the [Running Inside Gnome](https://github.com/qtile/qtile/blob/master/docs/manual/config/gnome.rst) page and kept getting issues with the `dbus-send` command.  For my setup (Ubuntu 15.10 Wily, D-Bus 1.10.0), I just needed to change `--print-reply=string` to `--print-reply`.

Also, I see mention of extra configuration for `gnome-panel`, but haven't been able to find any details (or example setups).  If anyone knows where that information is, please tell me!
Otherwise, `gnome-panel` won't work from the command line; the panel will display, but it's non-responsive.  I can provide details if anyone is interested.

**Comment by EliahShure** [comment]
try `--print-reply=literal` instead.
Just installed `gnome-panel` to test and it works fine here.

If some settings stop working after that (e.g. Displaysettings) u can also try adding `DesktopName=Unity` to the end of `usr/share/gnome-session/sessions/qtile.session` and/or `gnome-panel` to the line starting with `RequiredComponents=` in the same file.

**Comment by brandonwillard** [comment]
Ah, yes, I tried `--print-reply=literal`, too, but it seems like `gnome-panel` is unresponsive for completely unrelated reasons.  I'm gonna bet that it's my local gnome/session setup, like you implied.  I've tried every different type of session file settings I could find, but nothing seems to kick off `gnome-panel` right after login via LightDM (`qtile` will start just fine, though) and--as before--if I startup `gnome-panel` manually it simply won't respond to mouse clicks (or anything for that matter).
Again, I'm pretty sure this is a personal problem, so no need to keep this issue open.
